src/studentView.pug

```diff
@@ -0,0 +1,53 @@
+#studentView(hidden='')
+    header
+        #date
+        #account-icon
+            img(src='./gui/img/User.png' alt='User Icon')
+            span#account-username-display Welcome, User
+            #logout
+                span Logout
+    .node-search(ondragover='allowDroppingNode(event)' ondrop='deleteNode(event)')
+        #search-settings
+            input(type='search')
+            span.radioGroup
+                input#search-current(type='radio' name='search-where' value='Search in current module')
+                label(for='search-current') Search Current
+            span.radioGroup
+                input#search-all(type='radio' name='search-where' value='Search in all modules')
+                label(for='search-all') Search All
+        #search-results
+            button.accordion Arithmetic
+            .panel
+                #addition.node.arithmetic.addition(draggable='true' ondragstart='pickUpNode(event)')
+                    input(type='number')
+                    p
+                    input(type='number')
+                #subtraction.node.arithmetic.subtraction(draggable='true' ondragstart='pickUpNode(event)')
+                    input(type='number')
+                    p
+                    input(type='number')
+                #multiplication.node.arithmetic.multiplication(draggable='true' ondragstart='pickUpNode(event)')
+                    input(type='number')
+                    p
+                    input(type='number')
+                #division.node.arithmetic.division(draggable='true' ondragstart='pickUpNode(event)')
+                    input(type='number')
+                    p
+                    input(type='number')
+            button.accordion Algebraic Equations
+            .panel
+            button.accordion Graphing
+            .panel
+    .tab-switcher
+        ul
```

Comment:
The concept of a tabbed frame could be nice to have a template for as well. That way we can say that we want a tabbed frame and that these are the tabs and these are the views. This should be possible and would be convenient for later